tests/CppUTestExt/MockSupportTest.cpp

```diff
@@ -1720,24 +1750,39 @@ TEST(MockSupportTestWithFixture, mockExpectationShouldIncreaseNumberOfChecks)
     LONGS_EQUAL(4, fixture.getCheckCount());
 }
 
-#if !defined(__MINGW32__) && !defined(_MSC_VER)
+static bool cpputestHasCrashed;
+
+static void crashMethod()
+{
+    cpputestHasCrashed = true;
+}
 
 static void crashOnFailureTestFunction_(void)
 {
     mock().actualCall("unexpected");
 }
 
-TEST(MockSupportTestWithFixture, shouldCrashOnFailure)
+#include "CppUTestExt/OrderedTest.h"
+
+TEST_ORDERED(MockSupportTestWithFixture, shouldCrashOnFailure, 10)
 {
+    cpputestHasCrashed = false;
     mock().crashOnFailure();
-    fixture.registry_->setRunTestsInSeperateProcess();
+    UtestShell::setCrashMethod(crashMethod);
     fixture.setTestFunction(crashOnFailureTestFunction_);
     fixture.runAllTests();
-    fixture.assertPrintContains("Failed in separate process - killed by signal 11");
+    CHECK(cpputestHasCrashed);
+    UtestShell::resetCrashMethod();
+    mock().clear();
```

Comment:
So we change

`void crashOnFailure(void)` to `void crashOnFailure(bool)`, right?

BTW, for MockSupport, the TestTerminator is thrown away at some point, so the setting doesn't seem to survive too long. For MockSupport_c, on the other hand, it causes real trouble because the test terminator is static.